integration_test/install_test.go

```diff
@@ -105,6 +109,115 @@ func TestKrewInstall_Manifest(t *testing.T) {
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
 
+func startLocalServer(t *testing.T) (*httptest.Server, string) {
+	t.Helper()
+
+	server := httptest.NewUnstartedServer(http.FileServer(http.Dir("testdata")))
+
+	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	if err != nil {
+		t.Fatalf("trouble starting local server")
+	}
+	server.Listener = listener
+	server.Start()
+	return server, server.URL
```

Comment:
You can probably just do this:

```
httptest.NewServer(..)
return server.URL, server.Close
```

it does everything you want, and then you can embed it to the test because it's 2 lines.